src/styles/_bind-service.less

```diff
@@ -52,23 +52,49 @@
 }
 
 .results-message {
-  line-height: 1.4;
-  margin-top: 0;
+  flex: 1 1 0%; // IE
   padding-left: 10px;
-  strong {
-    .word-break();
+  @media(min-width: @screen-sm-min) {
+    padding-left: 15px;
   }
+  h3,
+  .h3 {
+    font-size: 15px;
+    line-height: 1.4;
+    margin-bottom: 0;
+    margin-top: 0;
+    @media(min-width: @screen-sm-min) {
+      font-size: 20px;
+      font-weight: 300;
+    }
+    strong {
+      font-weight: 600;
+      .word-break();
+    }
+  }
+}
```

Comment:
Seems odd to me to use h3 and then override almost all the values. Does this have to be an h3?

Reply:
The existing node was an h3, and I didn't change that.  I believe it should be a heading of some value so it has semantic value (probably an h1).  It looks like you originally authored it as an h3.  Any insight as to why?  Heading styles can and should be overridden.  Semantics !== style.

Reply:
I don't recollect why it was an h3. I'm ok with this as is. Thanks.